composer.json

```diff
@@ -14,5 +15,10 @@
         "psr-4": {
             "Neos\\RedirectHandler\\": "Classes"
         }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.0.x-dev"
+        }
```

Comment:
@kitsunet: may I ask why? personally would prefer just having one branch when the only difference is the composer requirement